src/style.css

```diff
@@ -0,0 +1,189 @@
+/* #region Reset */
+:root {
+  box-sizing: border-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  --Main-accent: #060b35;
+  --Secondary: #616070;
+  --Blue-accent: #00acdc;
+
+}
+
+body {
+  margin: 0;
+}
+/* #endregion */
+/* #region Card */
+.card {
+  position: relative;
+  left: 0;
+  top: 0;
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__product-photo {
+  position: absolute;
+  left: 19px;
+  top: 32px;
+  width: 160px;
+  height: 134px;
+  background-image: url("images/imac.jpeg");
+  background-position: center;
+  background-size: 160px 134px;
+}
+
+.card__product-title {
+  text-decoration: none;
+  position: absolute;
+  width: 166px;
+  left: 16px;
+  top: 206px;
+  color: #060b35;
+}
+
+.card__product-code {
+  position: relative;
+  width: 98px;
+  height: 14px;
+  top: 246px;
+  left: 16px;
+
```

Comment:
1. position is also unnecessary, use margins
2. deafault width value is fit-content here, so it will fit your text inside, so you can also remove width here